packages/core/src/constants.ts

```diff
@@ -26,6 +26,7 @@ const AWS_LAMBDA_CONFIG = {
   region: '',
   cf_distribution_id: '',
   lambda_arn: '',
+  routes_for_invalidation: ['/'],
```

Comment:
The type is extracted from this, not sure why do it like that tho. In theory this could also be `routes_for_invalidation: ['']` which outputs the same.